nbformat/v4/nbformat.v4.schema.json

```diff
@@ -64,6 +64,19 @@
                     "description": "The title of the notebook document",
                     "type": "string"
                 },
+                "description": {
+                    "description": "A summary of the notebook",
+                    "type": "string"
+                },
+                "tags": {
```

Comment:
I do think it makes sense to have a notion of notebook tags. Does this spec match that of the cell tags?

Reply:
Yeah, I almost re-used the xref it until I saw the description for the cell tags included mention of cells themselves and the purpose.